service/application/src/main/java/com/solace/maas/ep/event/management/agent/processor/RouteCompleteProcessorImpl.java

```diff
@@ -23,18 +21,8 @@ public RouteCompleteProcessorImpl(ScanStatusService scanStatusService) {
     @Override
     public void process(Exchange exchange) throws Exception {
         exchange.getIn().setHeader(RouteConstants.SCAN_STATUS, ScanStatus.COMPLETE);
-
         String scanId = (String) exchange.getIn().getHeader(RouteConstants.SCAN_ID);
-
```

Comment:
I pulled this code into a shared abstract class called `RouteStateProcessor` that's used by `RouteCompleteProcessor` and `RouteFailedProcessor`.